sdk/tests/conformance2/sync/sync-webgl-specific.html

```diff
@@ -77,11 +78,99 @@
   shouldBe("gl.getError()", "gl.NO_ERROR");
   gl.clientWaitSync(sync, 0, max + 1);
   shouldBe("gl.getError()", "gl.INVALID_OPERATION");
+
+  requestAnimationFrame(runGetSyncParameterTest);
+}
+
+function runGetSyncParameterTest() {
+  debug("");
+  debug("Verifying sync object isn't signaled too early");
+  sync = gl.fenceSync(gl.SYNC_GPU_COMMANDS_COMPLETE, 0);
+  // Verify as best as possible that the implementation doesn't allow a sync
+  // object to become signaled in the same frame, by spin-looping for some time.
+  var startTime = Date.now();
+  while (Date.now() - startTime < 2000) {
+    gl.flush();
+    gl.finish();
+    if (gl.getSyncParameter(sync, gl.SYNC_STATUS) == gl.SYNC_SIGNALED) {
+      testFailed("Sync object was signaled too early");
+      finishTest();
+      return;
+    }
+  }
+  testPassed("Sync object wasn't signaled too early");
+  startTimeOfFinish = Date.now();
+  requestAnimationFrame(finishSyncParameterTest);
+}
+
+var startTimeOfFinish = 0;
+
+function finishSyncParameterTest() {
+  if (startTimeOfFinish == 0) {
+    startTimeOfFinish = Date.now();
+  }
+  if (Date.now() - startTimeOfFinish > 2000) {
```

Comment:
2000 used without being a named constant again. Please name and define one.

Reply:
Thanks for your feedback. Named and defined one.